### PR TITLE
Hint about other files on the lore page

### PR DESCRIPTION
**Pull request by NetSysFire**
#### Summary
None

#### Purpose of change

While digging for lore in the newspapers I thought it might be helpful to mention the newspapers in the lore document. I also found a mention about bees getting bigger (from #43288), which may be a light-hearted addition since bees getting bigger is basically an external running gag but I decided to repurpose it into serious lore.

#### Describe the solution

I decided to make the month of appearance march since this is when bees have their first spike of activity or something.

#### Describe alternatives you've considered

Adding giant fuzzy pacifist bumblebees just because.

#### Testing

n/a

#### Additional context

I am annoyed about missing lore and decided I will partially take matters into my own hands (also see the living world project: https://github.com/orgs/CleverRaven/projects/5).

### CODE REVIEW

**Review by Maleclypse — DISMISSED** [review]
Ties together details in a more concrete fashion.

File: lore-background.md [hunk 1]
```diff
@@ -66,6 +67,7 @@ Note: Canon date of game start is in "YEAR", which equals the IRL year the game
 - **December 18**: Sample XE-037 is brought back to Earth from the netherum. It is about thirty grams of blob matter. Standard decontamination protocols are observed. However, standard protocols do not take into account the possibility of malicious, intelligent molecular-scale contaminants. Microscopic particles of XE-037 migrate from the contamination filters onto the skin of scientists and escape containment. Soon after, the Blob enters the groundwater and begins to spread at extremely low concentrations.
 
 #### Year -1
+- **March**: The first effects of the blob contamination become visible as bees two or even three times their size are sighted. They are non-aggressive and appear to cause more good than harm due to increased honey production.
```
Comment (Fris0uman) [inline lore-background.md:70]: I don't think bigger bees would make more honeys, actually bigger bees would probably need more pollen which would either be impossible to aquire enough to sustain themselves or could lead to a significant decrease in biodiversity as the giant bees overcompete every other pollinating insects.

Comment (I-am-Erk) [reply lore-background.md:70]: That is actually meant as an unreliable narration. Bigger bees would be terrible.

Comment (Fris0uman) [reply lore-background.md:70]: So not a thing to inscribe in the design doc as being actually true

Comment (kevingranade) [inline lore-background.md:70]:
```suggestion
```

**Pull request merged**